### PR TITLE
Add option to disallow printing for deceased

### LINKED ISSUE #440: Change statement to allow selection of no printing if patient is deceased.
Change statement to allow selection of no printing if patient is deceased. This should be user definable using a global variable.

### PR DESCRIPTION
**Pull request by azwreith**
Resolves #440 

Lines 1127-1133 of `globals.inc.php` & 107,451 of `statement.inc.php`
Couldn't really test since I was having difficulty adding payments, but I think it should work :pray: 

Thanks!

### CODE REVIEW

File: sites/default/statement.inc.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
  *
  *  2.  Branded Statement, whose core is build from 1., the original statement, using HTML2PDF.
  *
- *      To customize 2., add your practice location/images/practice_logo.gif
```
Comment (aethelwulffe) [inline sites/default/statement.inc.php]: This should be practice_logo.png...I think.

Comment (aethelwulffe) [inline sites/default/statement.inc.php]: White space is making review very difficult...

File: sites/default/statement.inc.php [hunk 2]
```diff
@@ -442,6 +447,9 @@ function create_statement($stmt) {
  #minimum_amount_to _print
  if ($stmt[amount] <= ($GLOBALS['minimum_amount_to_print']) && $GLOBALS['use_statement_print_exclusion']) return "";
 
+ // Don't print if decreased
+ if ($GLOBALS['disallow_print_deceased']) return "";
```
Comment (aethelwulffe) [inline sites/default/statement.inc.php]: This should probably give a message as to WHY it is not printing...."Your patient is DEAD.  You must start a new game or load a saved one!"

Comment (teryhill) [inline sites/default/statement.inc.php]: if you are printing 100 statements do you want a statement printed with "Your patient is DEAD. You must start a new game or load a saved one!" stuck some where in it?

**Comment by aethelwulffe** [comment]
Log output window, interface warning, print the statement with "Patient Deceased: Printing of statements for Deceased Patients disabled in ADMINISTRATION->Globals." or something other than simply not printing the statement.  I can just see the pissed off office worker trying over and over to print the same statement...not understanding what the issues is.  "I have been printing statements for this patient for 50 years!  Why doesn't it work now!!!!"
  If you guys are OK with the behavior as-is, I am cool with that.  If I am describing an edge case you don't think is worth addressing and all that.  Just giving my input.

File: sites/default/statement.inc.php (hunk 1 above)
Comment (azwreith) [inline sites/default/statement.inc.php]: This was there from before. I'll push a commit changing that!

Comment (azwreith) [inline sites/default/statement.inc.php]: Sorry about the whitespace removal thingy, my editor (Atom) does it automatically. I shall look into disabling that for this project. Or maybe we can create an issue out of this and removal redundant whitespace from every file in a single PR

**Comment by azwreith** [comment]
@aethelwulffe Sure, I can do that! And it makes sense too yep. Though if we are doing for this, we might as due it for other cases such as it not printing a statement when below the minimum amount, etc. What do you think?

**Comment by aethelwulffe** [comment]
https://github.com/atom/whitespace/issues/10
Terry, as the originator of this issue, should have final say in the needed implementation.  He is, after all, dealing with the issue as a specific requirement from an actual user.

**Comment by azwreith** [comment]
Thanks for the link! I'll disable it for now. I shall figure out how to set it on a per project basis later on ^^"
Though I think we really should run the project files though a whitespace stripper script once if possible, but that is a seperately issue on its own

**Comment by aethelwulffe** [comment]
I am withdrawing the change request for a notice that you are printing for deceased.  This has two reviews, and has been here for weeks, so I am going to merge it.
Good job @azwreith

**Pull request merged**